package.json

```diff
@@ -12,6 +12,7 @@
 		"@fortawesome/free-regular-svg-icons": "^6.2.1",
 		"@fortawesome/free-solid-svg-icons": "^6.2.1",
 		"@fortawesome/react-fontawesome": "^0.2.0",
+		"@heroicons/react": "^2.0.13",
```

Comment:
Do you need all these libraries for icons? Maybe one of the libraries has all the icons you need?